src/ddmd/expression.d

```diff
@@ -8772,7 +8773,7 @@ extern (C++) final class DotIdExp : UnaExp
                     if (v.type.ty == Terror)
                         return new ErrorExp();
 
-                    if ((v.storage_class & STCmanifest) && v._init)
+                    if ((v.storage_class & STCmanifest) && v._init && !wantsym)
```

Comment:
There might simply be different folks (or brains). I'm having problems working with variables that consist of more than 2 nouns as well. While I do read a lot, trying to read prose source code always leaves me dazed. Might be some interlink congestion between the wordy and the logical part of the brain ;).